pkg/ebpf/testdata/teardown_dnat.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -x
+
+# tear down the testing interface, and iptables rule
+ip link del dummy0
+iptables -t nat -D OUTPUT -d 2.2.2.2 -j DNAT --to-destination 1.1.1.1
+
+# clear out the conntrack table
+conntrack -F
```

Comment:
From the PR description:
> Moved some required test scripts for pkg/ebpf into its testdata folder. This makes the package tests self-contained rather than requiring on another package's test scripts.

Reply:
> Moved

It looks like setup/teardown DNAT scripts are copied into both testdata folders. Is having isolated test data worth the duplication?

Reply:
Sorry, copied would be the correct term. I was compiling test binaries and running them on other hosts/VMs and the relative paths into other packages was causing problems, hence why I made this change. Ideally, these would not be scripts at all, but Go code so that we could copy a compiled test binary around by itself.